integration/generic-metadata/some-file.ts

```diff
@@ -0,0 +1,3 @@
+export interface Foo {
+  foo: "bar";
```

Comment:
Is this an extra file that just snuck in?

Reply:
Yep. I added that file because `some-file` is directly referenced in the generated code but was not provided. This bug was picked up by the `tsc:check` script.